src/app/settings/page.jsx

```diff
@@ -1,20 +1,40 @@
+'use client';
 import userData from './settingsInfo.json';
 import UserCardWrapper from '@/components/User/UserCardWrapper';
 import SavedSearchList from '@/components/User/SavedSearchList';
 import AddNewPreferenceButton from '@/components/User/PreferenceButton';
 import { Box } from '@mui/material'; 
+import { useState, useEffect } from 'react';
 
 export default function Page() {
+  const [userData, setUserData] = useState(null);
+
+  useEffect(() => {
+    fetch('/api/users')
```

Comment:
we fetch user in 3 different places now, I've created a ticket to fetch it in Redux and reuse, can be as is in this PR